### PR TITLE
atbash exercise added

### PR DESCRIPTION
**Pull request by elorest**

### CODE REVIEW

**Comment by mhelmetag** [comment]
Awesome! Also in response to your question about when this will show up in exercism... this isn't an active track yet. We need 10 exercises. Almost there!

**Comment by elorest** [comment]
@mhelmetag My question was more specifically about why these new ones should show up instead of just the 7? With the 3 I've provided we should be at 10 now though :)

**Comment by tejasbubane** [comment]
Ah, you mean the [languages page](http://exercism.io/languages). If I am not wrong this is because `xapi` is not yet updated http://x.exercism.io/v3/tracks/crystal/todo. Which is because the submodule in https://github.com/exercism/x-api is not yet updated.

File: config.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "bob",
     "raindrops",
     "leap",
-    "pangram"
+    "pangram",
+    "atbash_cipher"
```
Comment (kytrinyx) [inline config.json]: This needs to be `atbash-cipher` (hyphen, not underscore)

File: exercises/atbash_cipher/atbash_cipher_spec.cr [hunk 2]
```diff
@@ -0,0 +1,40 @@
+require "spec"
```
Comment (kytrinyx) [inline exercises/atbash_cipher/atbash_cipher_spec.cr]: Here too—the directory needs to match the slug, `atbash-cipher`.

**Comment by kytrinyx** [comment]
Sorry—yes, this is my fault. I've had a bit of a hectic week. Deploying now.

**Pull request merged**